### ISSUE TITLE
Stretch Goal: Directions

### ISSUE THREAD

**Issue by hi-michelleprimiani** [NSS-Day-Cohort-66/holiday-road-plm#46]
Only after the main fetures of the application listed above are complete, you can work on the stretch goal of providing directions. For this feature, you will be using the Graphhopper API.

Once the user has saved an itinerary, and it is listed on the aside bar, refactor your application to add a Get Directions button to each HTML representation of the itinerary. When the user clicks that button for an itinerary, the user should be presented with step-by-step instructions for the trip. It should include

All 4 locations (Nashville, bizarrerie, eatery, and national park) need to be sent to Geocoding API to the the latitude and longitude for each one.

Then all 4 lat/long pairs should be in the URL for the request to the Routing API.

The step-by-step instructions in the response from the Routing API should be displayed below the Itinerary Preview section.
